Chandra.taco/Chandra/taco/version.py

```diff
@@ -9,7 +9,7 @@
 restrictions therein are applicable.
 """
 
-version = '0.2.1'
+version = '0.2.2'
```

Comment:
The convention for making a package Py3-compatible is to bump the major version to 3 and leave the rest, so `3.2.1`.